### PR TITLE
added web console availability SLO

### PR DESCRIPTION
**Pull request by thanasisk**

### CODE REVIEW

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json [hunk 1]
```diff
@@ -705,8 +713,8 @@
     ]
   },
   "time": {
-    "from": "now-30m",
-    "to": "now"
+    "from": "2018-10-09T10:11:34.650Z",
+    "to": "2018-10-09T12:11:34.662Z"
```
Comment (mjudeikis) [inline sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: These should be moving window as before. Not hardcoded date.

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json [hunk 2]
```diff
@@ -683,7 +687,11 @@
     "list": [
       {
         "allValue": null,
-        "current": {},
+        "current": {
+          "tags": [],
+          "text": "mjudeikist",
+          "value": "mjudeikist"
+        },
```
Comment (mjudeikis) [inline sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: where this came from? :)

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json [hunk 3]
```diff
@@ -369,6 +369,10 @@
       "steppedLine": false,
       "targets": [
         {
+          "expr": "http_requests_total{job=\"kubernetes-service-endpoints\",kubernetes_name=\"webconsole\",kubernetes_namespace=\"openshift-web-console\",status!~\"5..\"} / http_requests_total{job=\"kubernetes-service-endpoints\",kubernetes_name=\"webconsole\",kubernetes_namespace=\"openshift-web-console\"}",
```
Comment (mjudeikis) [inline sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: we need Prometheus alert rules and aggregated metric too. As separate PR potentially.

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json (hunk 2 above)
Comment (thanasisk) [reply sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: I had the view of your cluster open, so I guess it got saved into the JSON, please advise what to do.

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json (hunk 3 above)
Comment (thanasisk) [reply sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: I had in mind to do a separate PR for these, please advise.

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json (hunk 2 above)
Comment (jim-minter) [reply sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: revert that hunk please

**Comment by jim-minter** [comment]
lgtm once the unnecessary hunk has been reverted

File: sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json (hunk 3 above)
Comment (mjudeikis) [reply sre/grafana-deployment/provisioning/dashboards/openshift-cluster-slo.json]: let's make it different for simplicity.

**Comment by jim-minter** [comment]
unfortunately this is probably broken - see https://github.com/openshift/azure-misc/pull/98#issuecomment-428947875

**Pull request merged**